sdk/cloudmachine/Azure.Projects/src/Ofx/OfxProjectClient.cs

```diff
@@ -0,0 +1,43 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.ClientModel.Primitives;
+
+namespace Azure.Projects;
+
+/// <summary>
+/// The project client.
+/// </summary>
+public partial class OfxProjectClient : ProjectClient
```

Comment:
Did you intend to rename this?

Reply:
Yeah, but I don't have a good name idea.